WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsListFragment.java

```diff
@@ -306,6 +307,7 @@ private void showEmptyView(@StringRes int titleResId) {
         showEmptyView(titleResId, 0, 0);
     }
 
+    @SuppressLint("ResourceType")
```

Comment:
It looks like this lint error came from the `> 0` checks for the resource IDs. Could we update those to `!= 0` and do away with the lint suppression?